pilot/pkg/security/authz/model/principal.go

```diff
@@ -70,6 +70,7 @@ func principalAuthenticated(name *matcherpb.StringMatcher) *rbacpb.Principal {
 func principalSourceIP(cidr *corepb.CidrRange) *rbacpb.Principal {
 	return &rbacpb.Principal{
 		Identifier: &rbacpb.Principal_SourceIp{
+			// nolint: staticcheck
 			SourceIp: cidr,
```

Comment:
@yangminzhu are you tracking updating this?

Reply:
yes, this is updated in a follow PR in https://github.com/istio/istio/pull/27906